packages/core/test/financial-templates/common/FundingRateApplier.js

```diff
@@ -198,7 +198,7 @@ contract("FundingRateApplier", function(accounts) {
   });
 
   it("Proposal time checks", async () => {
-    const newRate = { rawValue: toWei("-0.0001") };
+    const newRate = { rawValue: toWei("-0.000001") };
```

Comment:
what happened here?

Reply:
The funding rate range is now constrained to [0.000001, -0.000001] so this rate had to be changed.